src/components/Navigation/index.tsx

```diff
@@ -70,7 +70,7 @@ const Navigation: React.FC<Props> = (props) => {
             }
             onClick={() => props.setMenuOpen(false)}
           >
-            Projektwoche
+            Fokuswochen
```

Comment:
I thought "Fokuswochen" made more sense, or should it be "Fokuswoche" @Maascha ?

Reply:
both are good :) at each point in time, there is at most one week,so I put Fokuswoche, but plural is not particularly wrong, so we can keep that change.